website/docs/components/icon-tile/partials/guidelines/overview.md

```diff
@@ -1 +1 @@
-IconTiles are used to represent objects and resources, either unrelated, or directly related to a product.
\ No newline at end of file
+Icon Tiles represent objects and resources unrelated or directly related to a product.
```

Comment:
I'm not sure if this line makes sense to me, it seems contradictory.

Reply:
It's saying the objects and resources don't need to be related to a specific product, but they can be. 

For instance, use the IconTile in the color "consul" to represent a Consul cluster. It's not the Consul brand IconTile, but an IconTile in a specific product color because the object relates to that product. 
`<Hds::IconTile @icon="cluster" @color="consul" />`

On the flip side, if the cluster isn't related to a specific product, or maybe it's related to multiple products (not sure how accurate this example is 🤷🏻‍♀️  don't come for me. lol) then it would use the default neutral color. 
`<Hds::IconTile @icon="cluster" />`

Does that make sense? Is there another way to represent this info in a less confusing, more concise way?